src/stories/containers/Finances/components/CardNavigationGeneric.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import React from 'react';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
+import type { PropsWithChildren } from 'react';
+
+interface Props {
+  className?: string;
+}
+
+const CardNavigationGeneric: React.FC<Props & PropsWithChildren> = ({ className, children }) => {
```

Comment:
istead of use a union type here, you can extends it in the props interface to avoid this. Would be:

```ts
interface CardNavigationGenericProps extends PropsWithChildren {
  className?: string;
}
```

Reply:
you are right, done